discord_bot.py

```diff
@@ -16,6 +16,10 @@
 import constants
 import scanner
 
+ERROR_EMOJI = "\U00002757"
```

Comment:
1. I wonder if these would work as the twitter emoji names instead like `':exclamation:'`
2. Alternatively, You can likely put the emoji directly since python supports utf8 `'❗'`
3. for consistency, use single quotes instead of double quotes